openfisca_web_api/controllers/entities.py

```diff
@@ -9,19 +9,8 @@
 from .. import contexts, conv, model, wsgihelpers
 
 
-def build_entity_data(entity):
-    entity_data = {'label': entity.label}
-    if entity.is_person:
-        entity_data['isPersonsEntity'] = entity.is_person
-    else:
-        entity_data.update({
-            'roles': entity.roles_description,
-            })
-    return entity_data
-
-
 @wsgihelpers.wsgify
-def api1_entities(req):
+def api2_entities(req):
```

Comment:
Judging from the route changes that were added after this comment, you mean the `v1` of `entities` had already been removed? Ok, then renaming was needed.